myproxy/source/ssl_utils.c

```diff
@@ -18,6 +18,10 @@
 #define EVP_PKEY_id(k) (k)->type
 #endif
 
+#ifndef MYPROXY_PRIVKEY_CIPHER()
```

Comment:
No I think it is good to put it here like this. If we ever want to define it (also) in the header file, this makes it more flexible.

One note: you need to drop the `()` in the `#ifndef`. This way it will give an `extra tokens at end of #ifndef directive` warning

Reply:
Ok, will do.